src/app.ts

```diff
@@ -50,7 +50,7 @@ app.enable("trust proxy");
 
 // parse body into req.body
 app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.urlencoded({ extended: true }));
```

Comment:
is this still needed?

Reply:
Agreed, i think this should revert to false as there's no need.